libcontainer/configs/network.go

```diff
@@ -50,23 +50,26 @@ type Network struct {
 	HairpinMode bool `json:"hairpin_mode"`
 }
 
-// Routes can be specified to create entries in the route table as the container is started
+// Route defines a route table entry.
```

Comment:
I think the proper term is "routing table" not "route table".

So
```suggestion
// Route defines a routing table entry.
```